src/rpc/methods/eth_sendUserOperation.ts

```diff
@@ -102,8 +103,8 @@ export async function addToMempoolIfValid(
         }),
         getUserOpValidationResult(rpcHandler, userOperation, entryPoint),
         rpcHandler.getNonceSeq(userOperation, entryPoint),
-        validatePvg(apiVersion, rpcHandler, userOperation, entryPoint),
-        rpcHandler.preMempoolChecks(userOperation, apiVersion),
+        boost ? [true, ""] : validatePvg(apiVersion, rpcHandler, userOperation, entryPoint),
```

Comment:
small nit, I would prefer to pass isBoost into validatePvg and return [true, ""] in the function if true